packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -449,7 +449,7 @@ class AttachDisk extends Component {
 
     const _isFreeForWriting = vdi =>
       vdi.$VBDs.length === 0 ||
-      some(vdi.$VBDs, id => {
+      every(vdi.$VBDs, id => {
```

Comment:
Were you able to test this?

Reply:
Yes, I have already tested this.
Test:  VDI has two VBDs and one between them is attached.
Result:  - `_isFreeForWriting`  with `some` gets `true`, the mode wiil be `RW`  which is not true. 
            -  `_isFreeForWriting`  with `every` gets  `false` and mode will be `RO`.